SU2_CFD/include/variables/CIncEulerVariable.hpp

```diff
@@ -55,6 +55,9 @@ class CIncEulerVariable : public CFlowVariable {
     inline IndexType ThermalConductivity() const { return nDim+6; }
     inline IndexType CpTotal() const { return nDim+7; }
     inline IndexType CvTotal() const { return nDim+8; }
+
+    /*--- For compatible interface with NEMO. ---*/
+    inline IndexType Temperature_ve() const { return std::numeric_limits<IndexType>::max(); }
```

Comment:
I need to 
```c++
#include <limits>
```
for this to work using gcc/11.1.0 + openmpi/4.1.2. as well as 11.2.0 + 4.0.5. The test-builds pass(ed) so it cannot be a major issue. 
[This stackoverflow post](https://stackoverflow.com/questions/4798936/numeric-limits-was-not-declared-in-this-scope-no-matching-function-for-call-t) gives resource that the limits header is used much less widely in gcc11 standard library headers -> requiring to explicitly include it.

And I guess we want to assure foward-compatibility ;)